app/templates/student/my_profile.html

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
This file should have been deleted... It's off my computer lol why is it still here?

Reply:
Removed it for you, you need to make sure to git add this file in order to register that you've completely removed it.